flutter/lib/src/binding_wrapper.dart

```diff
@@ -114,16 +138,21 @@ mixin SentryWidgetsBindingMixin on WidgetsBinding {
   void handleDrawFrame() {
     super.handleDrawFrame();
 
-    try {
-      final endTimestamp = _clock?.call();
-      if (_startTimestamp != null &&
-          endTimestamp != null &&
-          _startTimestamp!.isBefore(endTimestamp)) {
-        _frameTimingCallback?.call(_startTimestamp!, endTimestamp);
-      }
-    } catch (_) {
-      if (_options.automatedTestMode) {
-        rethrow;
+    if (_isTrackingActive && isFramesTrackingInitialized()) {
+      try {
+        _stopwatch?.stop();
+        if (_stopwatch!.elapsedMilliseconds >
```

Comment:
If `_stopwatch` is null here we'd crash. Should assign to local var and do null check.